src/common/docker-compose/converter.ts

```diff
@@ -123,7 +123,9 @@ export class ComposeConverter {
           }
         }
       } else {
```

Comment:
Since this block is using specific logic and not just setting what's passed in to the build args anymore, we should do something like 

```ts
else if (compose_build.args instanceof Object && Object.keys(compose_build.args).length)
```

in order to make sure that the new code will actually be passed a useful dictionary